### ISSUE TITLE
Does this work for IntelliJ Ultimate 2020.3?

### ISSUE THREAD

**Issue by RahulSDeshpande** [osvax/ide-eval-resetter#1]
Hey cheers!!
I tried using this plugin for `IntelliJ Ultimate 2020.3` but its not working.

**Comment by osvax** [comment]
Hi! I myself use phpshtorm, but I just downloaded and tested it on purpose. Everything works great. Before installing this plugin, please activate the trial mode. Then through the menu File -> Settings -> Plugins install ide-eval-resetter

**Comment by RahulSDeshpande** [comment]
Yes I am using trial mode, but somehow IntelliJ is detecting that I am using this plugin.

**Comment by RahulSDeshpande** [comment]
Thanks, let me try.

**Comment by SofianeDmt** [comment]
Hello friend, work in V2021.1 ? Thank for you job!

**Comment by gercoruy** [comment]
It's old this version try this one:
https://gitee.com/pengzhile/ide-eval-resetter

**Comment by XibalbaM** [comment]
yes it's work for the V2021.2.1 for me

**Comment by srifaquat** [comment]
@osvax  It is not working on V2021.2.1

**Comment by srifaquat** [comment]
@osvax It worked with latest jar you have provided thank you!!!
